app/dashboard/views.py

```diff
@@ -3555,10 +3548,19 @@ def verify_user_facebook(request):
             'ok': False,
             'message': 'Invalid code',
         })
-
+    
+    facebook_data = r.json()
+    check_facebook_id = Profile.objects.filter(facebook_id = facebook_data['id'])        
```

Comment:
fyi you can just query the JSON data without the need for an extra id here; `.filter(facebook_data__id=id)`

Reply:
I think you mean `identity_data_facebook__id` by `facebook_data__id`, right?
Isn't make it slow to query on JSON?